README.org

```diff
@@ -59,6 +59,12 @@ cabal new-build your-package-name
 cabal new-repl your-package-name:library:your-package-name
 #+end_src
 
+** Cache
+
+CI pushes to [[https://cachix.org][cachix]] so you can benefit from the cache if you pin a combination of =haskell.nix= and =nixpkgs= built by CI.
```

Comment:
```suggestion
CI pushes to [[https://cachix.org][cachix]] so you can benefit from the cache if you pin a combination of =haskell.nix= and [=nixpkgs= built by CI](https://github.com/input-output-hk/haskell.nix/blob/master/ci.nix#L8).
```

Reply:
I was actually hoping to add more substantial info about how to find a good combination but I don't know who I can ask.